src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyPressed);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyPressed);
+  }
+
+  keyPressed = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
```

Comment:
let's check `event.code` to check if the key is `space` and write `[space]` if so
<img width="200" alt="image" src="https://user-images.githubusercontent.com/55802630/182805355-70a09a98-9b2d-41e5-b658-4f54a5a73faa.png">